packages/js/src/classic-editor/initial-state.js

```diff
@@ -34,10 +34,8 @@ export const getInitialPostState = () => ( {
 	},
 	form: {
 		seo: {
-			title: dom.getPostSeoTitle() ||
-				window.wpseoScriptData.metabox.title_template_no_fallback ||
-				window.wpseoScriptData.metabox.title_template,
-			description: dom.getPostMetaDescription() || window.wpseoScriptData.metabox.metadesc_template,
+			title: dom.getPostSeoTitle(),
+			description: dom.getPostMetaDescription(),
```

Comment:
The fallback values that are taken from the templates are removed from the store, since we don't want to save the templates in the store.